modules/HapticGlove_module/src/GloveWearable.cpp

```diff
@@ -334,6 +334,7 @@ bool GloveWearableImpl::getFingertipPoseValues(Eigen::MatrixXd& values)
 
 bool GloveWearableImpl::setFingertipForceFeedbackValues(const std::vector<int>& values)
 {
+    return true;
```

Comment:
As discussed F2F, it would be nice to have a parameter to turn this on and off, rather than exiting prematurely. @RiccardoGrieco @lrapetti do you remember why this was needed?